flink-dist/src/main/assemblies/opt.xml

```diff
@@ -75,14 +75,6 @@
 			<fileMode>0644</fileMode>
 		</file>
 
-		<!-- ML -->
-		<file>
-			<source>../flink-libraries/flink-ml/target/flink-ml_${scala.binary.version}-${project.version}-jar-with-dependencies.jar</source>
-			<outputDirectory>opt/</outputDirectory>
-			<destName>flink-ml_${scala.binary.version}-${project.version}.jar</destName>
-			<fileMode>0644</fileMode>
-		</file>
-
```

Comment:
Why removing it from opt ? Then where can user find it in the binary distribution ?

Reply:
It wouldn't be part of the binary distribution a users would simply bundle it with their jar.

Reply:
Why would it not a part of the binary distribution ? What if user want to try ml in scala-shell or user want to run ml examples via bin/flink ? (currently there's no ml example in flink-examples, but I think we should add that in future)

Reply:
I brought up that very point in the JIRA. The reasons for doing are listed in the PR description.

The user would have to build flink-ml himself and put it into /lib. The scala-shell may be removed in the future given that it is currently broken for scala 2.12. The same applies to flink-ml since it is virtually frozen; instead we'd opt for users manually adding it to the distribution.

In fact I'd very much like to transition to a model where users add the things they need to flink-dist themselves. There's little benefit to having everything in flink-dist; why bundle all reporters/libraries etc. if only a 1 or 2 of them are used anyway.